packages/vuetify/src/components/VList/VListItem.tsx

```diff
@@ -294,13 +294,13 @@ export const VListItem = genericComponent<VListItemSlots>()({
           )}
 
           <div class="v-list-item__content" data-no-activator="">
-            { hasTitle && (
+            { hasTitle && !slots.default && (
               <VListItemTitle key="title">
                 { slots.title?.({ title: props.title }) ?? props.title }
               </VListItemTitle>
             )}
 
-            { hasSubtitle && (
+            { hasSubtitle && !slots.default && (
```

Comment:
What is the reason for this change?

Reply:
Actually. I have no real clue. This is something I just copied over from the other PR. It came with this commit, which I think is not really related to this change: https://github.com/vuetifyjs/vuetify/commit/e6792c1b79b0e0a5cab35db6720dd5b62dfc2d0a

If it is not needed or is bad practice, I will remove it.